testing_and_setup/compass/ocean/scripts/plots/plot_initial_state.py

```diff
@@ -65,45 +67,60 @@ def main():
     txt = txt + ' {:9.2e}'.format(np.min(var)) + \
         ' {:9.2e}'.format(np.max(var)) + ' ' + varName + '\n'
```

Comment:
I didn't have the energy to change all the formatting, but I would not use `str` addition like this.  For example, here, I would do:
```suggestion
    txt = '{}{:9.2e} {:9.2e} {}\n'.format(txt, np.amin(var), np.amax(var), varName)
```
I think `amin` and `amax` are considered to be the better than the aliases `min` and `max` because it clarifies that you want the min/max over the array vs. `minimum` and `maximum`, which give the element-wise min/max between two arrays.